packages/clerk-js/src/ui/elements/ApplicationLogo.tsx

```diff
@@ -29,9 +29,11 @@ type ApplicationLogoProps = PropsOfComponent<typeof Flex>;
 export const ApplicationLogo = (props: ApplicationLogoProps) => {
   const imageRef = React.useRef<HTMLImageElement>(null);
   const [loaded, setLoaded] = React.useState(false);
-  const { logoImageUrl, applicationName, homeUrl } = useEnvironment().displayConfig;
+  const { logoImageUrl, applicationName, homeUrl, afterLogoClickUrl } = useEnvironment().displayConfig;
```

Comment:
Do we want to use this value that's set from the accounts portal on the components after all?

Reply:
Both @desiprisg, we want to change the ApplcationLogo from the accounts portal and components